src/main/java/com/kammatysiak/medicalclinic/controller/PatientController.java

```diff
@@ -0,0 +1,53 @@
+package com.kammatysiak.medicalclinic.controller;
+
+import com.kammatysiak.medicalclinic.model.dto.PasswordClassDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientCreateDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientDTO;
+import com.kammatysiak.medicalclinic.model.entity.Patient;
+import com.kammatysiak.medicalclinic.service.PatientService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/patients")
+public class PatientController {
+
+    private final PatientService patientService;
+//w controllerze definiujemy endpointy (punkty komunikacji) a serwis ma dostarczyc wlasciwe dane do kontrolera
+
+    @GetMapping
+    public List<PatientDTO> getPatients() {
+        return patientService.getPatients();
+    }
+
+    @GetMapping("/{email}")
+    public PatientDTO getPatient(@PathVariable String email) {
```

Comment:
@PathVariable("email") <- dzieki temu jesli zmienna string email zmieni swoja nazwe totak czy siak zadziala nam to prawidłowo poniewaz na sztywno mowimy, ze parametr przed ktorym jest ten pathVariable name = email to jest ten ze sciezki oznaczonej {email}